src/cli/cmds/init.ts

```diff
@@ -25,6 +25,7 @@ import {TwaGenerator} from '../../lib/TwaGenerator';
 import {TwaManifest} from '../../lib/TwaManifest';
 import {validateColor, validatePassword, validateUrl, notEmpty} from '../inputHelpers';
 import {ParsedArgs} from 'minimist';
+import {APP_NAME} from '../consts';
```

Comment:
Could you please renaming this file to "constants"?

Reply:
done